### DISCUSSION TITLE
The images added in the documentation appear much blurrier than the original ones

### DISCUSSION THREAD

**Discussion by dante1024** [docmost/docmost#574] [Help / Questions]
Dear project maintainer,
I hope this message finds you well. I'm writing to report two issues I encountered while using this project.
First, the images added in the documentation appear much blurrier than the original ones. I was wondering if the added images are compressed. Is there a way to make the original images display clearly?
Second, the "copy Link" function always fails. When I click "copy link", the link is not actually copied.
I really appreciate your efforts in developing this project and hope these issues can be addressed. Thank you so much.
Best regards,

**Comment by Philipinho** [comment]
Uploaded images are not compressed. We set the image width to 100% by default, which makes them larger than they ought to be on the editor. 
Ideally, it should be a fixed height and width. 
I have plans to work on this.

Your second issue; do you have https enabled? It is required for the copy function to work.
